gallery/src/components/NxDrawer/NxDrawerExample.tsx

```diff
@@ -0,0 +1,253 @@
+        cotton candy halvah. Fruitcake halvah I love cheesecake I love I love. Wafer sweet sweet roll apple
+        pie jelly-o cheesecake candy I love.
+      </NxP>
+      <NxP>
+        Brownie dessert candy wafer macaroon. Marzipan dragée liquorice biscuit icing I love.
+        Wafer pastry sweet candy canes pie pie icing <strong>brownie</strong>. Wafer jelly cake bear claw I
+        love caramels. Pie jelly-o candy jelly beans icing. Sweet gingerbread pastry jelly bonbon danish icing.
+      </NxP>
+      <NxP>
+        Toffee tootsie roll danish chupa chups dessert cookie I love gummi bears wafer. Dragée
+        chocolate cake cake brownie carrot cake caramels chocolate cake. Halvah jelly-o tart wafer
+        jelly. Pastry <em>carrot</em> cake cake bear claw bear claw. Toffee jelly beans biscuit jelly cotton
+        candy marzipan liquorice. Brownie candy pie sweet roll powder danish dragée. Cake cookie
+        topping pastry carrot cake donut. Bear claw danish powder muffin bonbon cookie. I love I love
+        tootsie roll croissant chupa chups oat cake. Icing gingerbread pastry marshmallow pudding
+        brownie gingerbread marzipan.
+      </NxP>
+      <NxP>
+        Dragée pastry soufflé shortbread donut fruitcake. Ice cream tart bear claw I love
+        cotton candy marzipan cotton candy cake danish. Pie gingerbread marshmallow bear claw
+        halvah tiramisu cotton candy icing topping. Liquorice chupa chups dessert carrot cake
+        macaroon wafer. Marshmallow apple pie danish muffin cupcake icing dessert I love lemon
+        drops. Cupcake I <em>love</em> candy canes dragée croissant cookie chocolate muffin. Marshmallow
+        powder lollipop cotton candy bonbon lollipop liquorice chupa chups jelly-o. Biscuit I
+        love marzipan pastry pie ice cream chocolate bar dessert sweet. Cake topping cookie
+        chocolate pie cupcake. I love pastry donut croissant macaroon chocolate cake icing macaroon marshmallow.
+      </NxP>
+      <NxP>
+        Lorem ipsum dolor sit amet, <strong>consectetur</strong> adipiscing elit,
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo{' '}
+        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum{' '}
+        dolore eu fugiat nulla pariatur.
+      </NxP>
+    </NxDrawer.Content>
+  );
+
+  return (
+    <>
+      <header className="nx-global-header">
+        <NxBackButton href="#/pages/Drawer" targetPageTitle="Documentation" />
+        <div className="nx-global-header__actions">
+          <NxButton title="Edit" variant="icon-only"><NxFontAwesomeIcon icon={faEdit} /></NxButton>
+        </div>
+      </header>
+      {showDrawer && (
+        <NxDrawer id="nx-drawer-simple"
+                  onCancel={() => setShowDrawer(false)}
+                  aria-label="Simple Drawer">
```

Comment:
Most real-world `NxDrawer`s will be labelled by their headers (using `aria-labelledby`), so that's the pattern I'd like our examples to demonstrate.

Reply:
> Most real-world `NxDrawer`s will be labelled by their headers (using `aria-labelledby`), so that's the pattern I'd like our examples to demonstrate.

Fixed here: 3ab6a3dcca04fee24a313c284f05560ec0529072